RingPublishingGDPR/src/main/java/com/ringpublishing/gdpr/RingPublishingGDPRShowConsentScreenListener.java

```diff
@@ -0,0 +1,17 @@
+package com.ringpublishing.gdpr;
+
+public interface RingPublishingGDPRShowConsentScreenListener
```

Comment:
a metoda "notifyConsentsUpdated()" jest w innym listenerze?

Reply:
jak rozumiem @adamszeremeta , masz na myśli to (z czym bym się zgadzał), by był 1 listener, a nie 2 osobne?

Reply:
może - nie wiem jak wygląda użycie w apce do końca ale pytanie czy z perspektywy Androida ma to sens

Reply:
Zrobiłem sobie na próbę też wersję z jednym listnerem, gdzie mamy 3 metody podobnie jak na iOS.

Takie rozwiązanie ma jednak kilka problemów.
- W Application używamy tylko jednej z tych metod a musimy przeciążyć wszystkie trzy.
- W SplashActivity używamy tylko 2 z 3 metod, czyli jedna też zostaje pusta
- Podczas sprawdzania zgód i potem już notyfikowania odpowiedniego listnera, jak mam 2 listnery to wiem, czy jestem nadal na splashu, bo czekam na odpowiedź. Jeśli już nie jestem na splashu to mój jeden listener jest nullem i wiem wtedy, że Ekran zgód muszę wyświetlić nad aplikacją niezależnie od tego czy już jestem na liście czy detalu. 
Tutaj bym musiał kombinować z jakimiś flagami.